packages/features/bookings/lib/sendBookingMessages.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
```

Comment:
@hariombalhara 
Rather than using custom code here, we can save the Slack webhook url to Webhooks when installing the app and allow the user to select the events (BOOKING_CREATED, BOOKING_RESCHEDULED, BOOKING_CANCELLED, MEETING_ENDED) for which they want to send messages to Slack while installing the app. Similar to the approach used for Make/Zapier.

Reply:
Yeah, we definitely need this.